spec/requests/api/requests_spec.rb

```diff
@@ -248,6 +248,31 @@
       expect(response.parsed_body).to match(expected)
       expect(response).to have_http_status(:ok)
     end
+
+    it "exposes allowed_tags in the request resources" do
```

Comment:
Both good, though my preference is on the API side since that is the intended usage and it tests both model and the API exposing it.